recipes-devtools/python/python-pymongo_3.2.2.bb

```diff
@@ -0,0 +1,13 @@
+DESCRIPTION = "Python driver for MongoDB"
```

Comment:
Commit is subsumed by other changes.

This is a valuable commit; this issue is however already addressed in
bmwcarit#509.